CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 * Order field mapping profile: Inactivate some fields when Order format is Physical or Other (UIDATIMP-1401)
 * Order field mapping profile: Inactivate Create Inventory fields when Order status is Open (UIDATIMP-1402)
 * Adjust mapping details for DI wrt PO prefixes and suffixes (UIDATIMP-1408)
+* Avoid private paths in stripes-core imports (UIDATIMP-1414)
```

Comment:
It should be moved to `## **6.1.0** (in progress)` or `## **6.0.3** (in progress)` (if it needs to be released in Orchid Bugfix)